lib/unleash/configuration.rb

```diff
@@ -87,6 +90,7 @@ def set_defaults
       self.environment      = 'default'
       self.url              = nil
       self.instance_id      = SecureRandom.uuid
+      @connection_id        = SecureRandom.uuid
```

Comment:
This is the dumbest comment I've ever left on a PR ever but the change in syntax here is a bit jarring. Any chance we can move this line either to the top of the bottom of this function?

Honestly I won't even be offended if you just ignore me

Reply:
It actually makes perfect sense to me. Had the same thought :) Moving it to the bottom

Reply:
Thank you! I appreciate you haha